memory-hog/prow_run.sh

```
@@ -34,5 +34,13 @@ cd $krkn_loc
 cat config/memory_hog_config.yaml
 cat $SCENARIO_FILE
 
-python3.11 run_kraken.py --config=config/memory_hog_config.yaml -o /tmp/report.out
+[ -z $JUNIT_TESTCASE ] && JUNIT_TESTCASE="memory-hog"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)